packages/container/lib/globals.module.scss

```diff
@@ -0,0 +1,13 @@
+/**
+ * IMPORTANT
```

Comment:
This warning is good, but I'm not super clear on what the `data-kz-component-defaults` is or where it should be used.

Should we be adding it to all our components? (And if so, should we do that in this PR?)

Should it be on the KaizenContainer only?

Is it strictly for internal use only?

Some guidance on when you should (and shouldn't) use this attribute would be nice.

Reply:
This probably ties into @makenosound's comment about being clear on what the boundaries of this component are.

What should the KaizenContainer do? What should be included here, and what needs to find a different home? Putting that info up front in the README would be good

Reply:
To be honest, I don't think I settled on that myself. We clearly need both global and component-level overrides. I think this PR solves global normalization and overrides well. 

I'd also like to solve component level overrides, but maybe this is not the right place for that. Ideally, there should be some kind of style utility for component-level defaults that would give the components the ability to opt-in, modify, extend the default component override as needed. However, we actively discourage consumers from extending Sass so I think a Sass class that consumers extend is a tacit endorsement of that approach. 

Admittedly, I think this approach is also kind of sucks.

Reply:
Decided to remove this completely. Component-level overrides should be done at the component-level.